pkg/kubeadm/master/pki.go

```diff
@@ -58,7 +58,7 @@ func newServerKeyAndCert(caCert *x509.Certificate, caKey *rsa.PrivateKey, altNam
 		return nil, nil, fmt.Errorf("unabel to create private key [%s]", err)
 	}
 	// TODO these are all hardcoded for now, but we need to figure out what shall we do here exactly
-	altNames.IPs = append(altNames.IPs, net.ParseIP("10.3.0.1"))
+	altNames.IPs = append(altNames.IPs, net.ParseIP("10.3.0.1"), net.ParseIP("10.16.0.1"))
```

Comment:
I guess the question is really whether service CIDR needs to be exposed as
an important flag, and whether we should do any guess work around the
default value for it.

On Tue, 6 Sep 2016, 18:23 Ilya Dmitrichenko, <errordeveloper@gmail.com>
wrote:

> This is one of the large todos that we need to tackle for the MVP phase.
>
> On Tue, 6 Sep 2016, 18:20 Lucas Käldström, <notifications@github.com>
> wrote:
>
>> In pkg/kubeadm/master/pki.go
>> <https://github.com/kubernetes/kubernetes/pull/31221#discussion_r77679299>
>> :
>>
>> > @@ -58,7 +58,7 @@ func newServerKeyAndCert(caCert *x509.Certificate, caKey *rsa.PrivateKey, altNam
>> >  		return nil, nil, fmt.Errorf("unabel to create private key [%s]", err)
>> >  	}
>> >  	// TODO these are all hardcoded for now, but we need to figure out what shall we do here exactly
>> > -	altNames.IPs = append(altNames.IPs, net.ParseIP("10.3.0.1"))
>> > +	altNames.IPs = append(altNames.IPs, net.ParseIP("10.3.0.1"), net.ParseIP("10.16.0.1"))
>>
>> Hmm, what's this?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/kubernetes/kubernetes/pull/31221/files/3e1a07e67fa739795052b9276098a1ef4fd532d9..00ca93345e50d6edfc2a736c97044465258a01ae#r77679299>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AAPWS48ugtfAJduYFmzta6P3Nb3-E7xIks5qnaDsgaJpZM4JqsTH>
>> .
>>
>